build/.vsts-ci.yml

```diff
@@ -50,29 +50,12 @@ stages:
         analyzeSecurity: false
         packageNugets: false
 
-- stage: UpdateTestEnvironment
-  displayName: 'Update Test Environment'
+- stage: UpdateCIEnvironment
```

Comment:
Does Deploy.yml just do a update of an existing environment?

Reply:
Yeah, it re-deploys the ARM template and also updates the web server like today